pkg/operator/resources_pod_template.go

```diff
@@ -207,14 +207,20 @@ func generatePodTemplate(
 		imagePathConfigReloader = *d.Agent.Spec.ConfigReloaderImage
 	}
 
+	boolFalse := false
+	boolTrue := true
 	operatorContainers := []core_v1.Container{
 		{
 			Name:         "config-reloader",
 			Image:        imagePathConfigReloader,
 			VolumeMounts: volumeMounts,
 			Env:          envVars,
 			SecurityContext: &core_v1.SecurityContext{
-				RunAsUser: pointer.Int64(0),
```

Comment:
@rfratto is there a specific reason why this was initially configured to run as root? do you see any problems with this?

Reply:
Probably just me being aggressive with copy and paste, I think it's safe to remove.